webpack/scenes/Content/Table/ContentTable.js

```diff
@@ -0,0 +1,118 @@
+import { getContent, getContentTypes } from '../ContentActions';
+import { selectContent, selectContentStatus, selectContentError, selectContentTypes, selectContentTypesStatus } from '../ContentSelectors';
+import SelectableDropdown from '../../../components/SelectableDropdown';
+import contentConfig from '../ContentConfig';
+import Loading from '../../../components/Loading';
+
+const ContentTable = () => {
+  const dispatch = useDispatch();
+  const status = useSelector(selectContentStatus);
+  const error = useSelector(selectContentError);
+  const response = useSelector(selectContent);
+  const enabledTypesResponse = useSelector(selectContentTypes);
+  const enabledTypesStatus = useSelector(selectContentTypesStatus);
+  const [contentTypeSelected, setContentTypeSelected] = useState('Python Packages');
+  const [searchQuery, updateSearchQuery] = useState('');
+  const [columnHeaders, setColumnHeaders] = useState([]);
+  const [contentTypes, setContentTypes] = useState(null);
+
+  const { results, ...metadata } = response;
+
+  /* eslint-disable react/no-array-index-key */
+  function buildRows(details) {
+    const rows = [];
+    if (details) {
+      columnHeaders.forEach((header, index) =>
+        rows.push(<Td key={index}>{header.getProperty(details)}</Td>));
+    }
+    return rows;
+  }
+
+  useEffect(() => {
+    dispatch(getContentTypes());
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  useEffect(() => {
+    if (contentTypes) {
+      setColumnHeaders(contentConfig().find(type =>
+        type.names.singular === contentTypes[contentTypeSelected][0]).columnHeaders);
+    }
+  }, [contentTypes, contentTypeSelected]);
+
+  useMemo(() => {
+    if (enabledTypesStatus === STATUS.RESOLVED) {
+      const types = {};
+      contentConfig().forEach((type) => {
+        if (enabledTypesResponse.includes(type.names.singular)) {
+          types[type.names.title] = [type.names.singular, type.names.plural];
+        }
+      });
+      setContentTypes(types);
+    }
+  }, [enabledTypesResponse, enabledTypesStatus]);
+
+  const fetchItems = useCallback(
+    params => getContent(contentTypes[contentTypeSelected][1], params),
+    [contentTypes, contentTypeSelected],
+  );
+
+  const emptySearchTitle = __('No matching content found');
+  const emptySearchBody = __('Try changing your search settings.');
+
+  if (enabledTypesStatus === STATUS.PENDING || !contentTypes) {
+    return <Loading />;
+  }
+
+  return (
+    <TableWrapper
+      {...{
+        metadata,
+        emptySearchTitle,
+        emptySearchBody,
+        searchQuery,
+        updateSearchQuery,
+        error,
+        status,
+        fetchItems,
+      }}
+      variant={TableVariant.compact}
+      autocompleteEndpoint={`/${contentTypes[contentTypeSelected][1]}/auto_complete_search`}
+      emptyContentTitle={`You currently don't have any ${contentTypeSelected}.`}
+      emptyContentBody={`${contentTypeSelected} will appear here when created.`}
```

Comment:
if you add `additionalListeners={[contentTypeSelected]}` to the Table here, it will update when content type is picked in the selector. That will be needed when we use this page for other content types..